data_generator/readers/tabulator_reader.py

```
@@ -82,12 +82,14 @@ def __load_row(row, asmt: Assessment, parse_asmt, parse_item):
         asmt.version = row['AssessmentVersion']
         asmt.year = int(row['AcademicYear']) + 1  # TODO - remove +1 when tabulator is fixed
         asmt.bank_key = row['BankKey']
-        asmt.overall_score_min = __getScore(row['ScaledLow1'])
-        asmt.overall_cut_point_1 = __getScore(row['ScaledHigh1'])
-        asmt.overall_cut_point_2 = __getScore(row['ScaledHigh2'])
-        asmt.overall_cut_point_3 = __getScore(row['ScaledHigh3'])
-        asmt.overall_cut_point_4 = None
-        asmt.overall_score_max = __getScore(row['ScaledHigh4'])
+
+        asmt_scale_scores = cfg.ASMT_SCALE_SCORE[asmt.subject][asmt.grade]
```

Tabulator has a bug where the scale scores aren't included for ICA's ... this uses them if present but falls back to hardcoded defaults.